tests/Aspire.MySqlConnector.Tests/ConformanceTests.cs

```diff
@@ -12,12 +13,12 @@
 
 namespace Aspire.MySqlConnector.Tests;
 
-public class ConformanceTests : ConformanceTests<MySqlDataSource, MySqlConnectorSettings>
+public class ConformanceTests : ConformanceTests<MySqlDataSource, MySqlConnectorSettings>, IClassFixture<MySqlContainerFixture>
 {
-    private const string ConnectionSting = "Host=localhost;Database=test_aspire_mysql;Username=root;Password=password";
-
-    private static readonly Lazy<bool> s_canConnectToServer = new(GetCanConnect);
-
+    private readonly MySqlContainerFixture _containerFixture;
+    private string ConnectionSting => RequiresDockerTheoryAttribute.IsSupported
+                                        ? _containerFixture.GetConnectionString()
+                                        : "Host=localhost;Database=test_aspire_mysql;Username=root;Password=password";
```

Comment:
This `private` property doesn't appear to be used; is it necessary?

Reply:
It is used in `PopulateConfiguration`.

Reply:
Right you are; sorry I missed that when reading through the file.